jcache/embedded/src/test/resources/infinispan_uri.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<infinispan xmlns="urn:infinispan:config:7.2">
```

Comment:
Could you also change the namespace version to `9.0` (which is being developed on `master branch`).